<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="4">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">#lineEdit_2 {background-color:gray;
border: 1px purple;
border-style: solid;
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;}

#lineEdit_2:focus { background-color:rgb(202, 255, 227);
border : 1px purple;
border-style: dashed;	
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="checkBox_10">
        <property name="text">
         <string>Adress</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_8">
        <property name="text">
         <string>Birthday</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="4">
       <widget class="QCheckBox" name="checkBox_9">
        <property name="text">
         <string>Year birthday</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="styleSheet">
         <string notr="true">#lineEdit_3 {background-color:gray;
border: 1px purple;
border-style: solid;
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;}

#lineEdit_3:focus { background-color:rgb(202, 255, 227);
border : 1px purple;
border-style: dashed;	
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>First Word</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="styleSheet">
         <string notr="true">#lineEdit_8 {background-color:gray;
border: 1px purple;
border-style: solid;
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;}

#lineEdit_8:focus { background-color:rgb(202, 255, 227);
border : 1px purple;
border-style: dashed;	
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
      <item row="9" column="4">
       <widget class="QLineEdit" name="lineEdit_10">
        <property name="styleSheet">
         <string notr="true">#lineEdit_10 {background-color:gray;
border: 1px purple;
border-style: solid;
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;}

#lineEdit_10:focus { background-color:rgb(202, 255, 227);
border : 1px purple;
border-style: dashed;	
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="styleSheet">
         <string notr="true">#lineEdit_5 {background-color:gray;
border: 1px purple;
border-style: solid;
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;}

#lineEdit_5:focus { background-color:rgb(202, 255, 227);
border : 1px purple;
border-style: dashed;	
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBox_7">
        <property name="text">
         <string>Pet Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">#lineEdit {background-color:gray;
border: 1px purple;
border-style: solid;
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;}

#lineEdit:focus { background-color:rgb(202, 255, 227);
border : 1px purple;
border-style: dashed;	
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="styleSheet">
         <string notr="true">#lineEdit_4 {background-color:gray;
border: 1px purple;
border-style: solid;
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;}

#lineEdit_4:focus { background-color:rgb(202, 255, 227);
border : 1px purple;
border-style: dashed;	
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string>Surname</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="styleSheet">
         <string notr="true">#lineEdit_6 {background-color:gray;
border: 1px purple;
border-style: solid;
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;}

#lineEdit_6:focus { background-color:rgb(202, 255, 227);
border : 1px purple;
border-style: dashed;	
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkBox_6">
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="checkBox_11">
        <property name="text">
         <string>Street</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Second Word</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBox_5">
        <property name="text">
         <string>Nickname</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="styleSheet">
         <string notr="true">#lineEdit_7 {background-color:gray;
border: 1px purple;
border-style: solid;
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;}

#lineEdit_7:focus { background-color:rgb(202, 255, 227);
border : 1px purple;
border-style: dashed;	
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
      <item row="10" column="4">
       <widget class="QLineEdit" name="lineEdit_11">
        <property name="styleSheet">
         <string notr="true">#lineEdit_11 {background-color:gray;
border: 1px purple;
border-style: solid;
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;}

#lineEdit_11:focus { background-color:rgb(202, 255, 227);
border : 1px purple;
border-style: dashed;	
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="styleSheet">
         <string notr="true">#lineEdit_9 {background-color:gray;
border: 1px purple;
border-style: solid;
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;}

#lineEdit_9:focus { background-color:rgb(202, 255, 227);
border : 1px purple;
border-style: dashed;	
border-radius: 7px;
padding: 0 2px 2px 2px;
selection-background-color: rgb(226, 155, 218);
font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox_13">
        <property name="text">
         <string>Add Simbols</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_12">
        <property name="text">
         <string>Add Number</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">#pushButton {
 color:#00c6ff;
 padding: 10px 20px;
 width:100px;
 text-decoration:none;
 text-align:center;
 margin:20px auto;
 display: block;
 background-image: linear-gradient(to left,transparent,transparent 50%,#00c6ff 50%,#00c6ff);
 background-position: 100% 0;
 background-size: 200% 100%;
 transition: all .25s ease-in;
 font: 400 18px tahoma;
 border: 1px solid #00C6FF;
}
#pushButton:hover {
background-position: 0 0;
color:green;
background-color: rgb(46, 52, 54);
 border: 2px solid #7922CC;
border-radius: 20px;
}</string>
        </property>
        <property name="text">
         <string>Get Start</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">#pushButton_2{
	color: white;
	background-color: rgb(0, 0, 0);
	border-radius: 7px;
	font: 16px
}

#pushButton_2:hover{
	color: green;
	font: 18px;
	height: 40px;
}</string>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDescription">
    <property name="title">
     <string>Description</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDescription"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
